### PR TITLE
i18n(es): Updated `middleware.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- Minor content fixes (broken links, typos, etc.)

#### Description

- What does this PR change? Give us a brief description.
Updated `middleware.mdx` based on this commit 22c58ac6580b01cfaa3005af0f494ced9f2fde8a

### CODE REVIEW

**Review by sasoria — APPROVED** [review]
LGTM ✅

File: src/content/docs/es/guides/middleware.mdx [hunk 1]
```diff
@@ -122,48 +121,41 @@ const orders = Array.from(Astro.locals.orders.entries());
...
+};
 ```
 
 ### Encadenamiento de middleware
 
-Múltiples middlewares pueden ser unidos, en un orden específico, utilizando [`sequence()`](#sequence):
+Múltiples middlewares pueden ser unidos en un orden específico utilizando [`sequence()`](#sequence):
 
 ```js title="src/middleware.js"
 import {sequence} from "astro/middleware";
```
Comment (sasoria) [inline src/content/docs/es/guides/middleware.mdx:142]: I think you can update this to `import { sequence }` from "astro/middleware";

**Comment by dreyfus92** [comment]
Thanks for your review @sasoria 😁

**Review by Yan-Thomas — APPROVED** [review]
LGTM!

**Pull request merged**